src/transaction/types/storage.rs

```diff
@@ -0,0 +1,76 @@
+use fuel_types::{bytes, Bytes32, Bytes64};
+#[cfg(feature = "random")]
+use rand::{distributions::Standard, prelude::Distribution, Rng};
+use std::cmp::Ordering;
+use std::io;
+
+pub const SLOT_SIZE: usize = Bytes64::LEN;
+
+#[derive(Debug, Default, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(
+    feature = "serde-types-minimal",
+    derive(serde::Serialize, serde::Deserialize)
+)]
+pub struct StorageSlot(Bytes32, Bytes32);
```

Comment:
Shouldn't we use named attributes? struct for tuples definition works very well for single elements. But for two or more, it becomes hard to read.

```suggestion
pub struct StorageSlot {
    key: Bytes32,
    value: Bytes32,
}
```